core/project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.5.5
+sbt.version=1.8.2
```

Comment:
Think we deleted this file in master. Do we need it?

Reply:
we probably need to rebase